core/src/main/java/org/apache/accumulo/core/conf/ClientProperty.java

```diff
@@ -178,6 +189,22 @@ public Long getLong(Properties properties) {
     return Long.parseLong(value);
   }
 
+  public Long getBytes(Properties properties) {
+    String value = getValue(properties);
```

Comment:
Could do `checkState(type == PropertyType.BYTES)`

Reply:
I had this check throwing an exception if it failed but it seemed unnecessary since a Number Format exception will get thrown.  Did you have any other idea how to handle a failure?

Reply:
This check would be in addition to the checks in `getValue`.  The main purpose of this check is to catch programmer error.  If I call `BATCH_WRITER_MAX_LATENCY_SEC.getBytes(props)` the code may work sometimes.

Reply:
Oh yeah good point, it won't always fail.  I will add them back.